web/src/tippyjs.js

```diff
@@ -105,6 +105,19 @@ export function initialize() {
         target: ".tippy-zulip-tooltip",
     });
 
+    delegate("body", {
```

Comment:
Added a separate tooltip configuration to manage the tooltip displayed when hovering over a User's profile card. Previously, it used `data-tippy-content`, so applying CSS to it could potentially break other tooltips.